### DISCUSSION TITLE
Is there a way to change language of a chart from Preview via API?

### DISCUSSION THREAD

**Discussion by sam-e-ch** [visualize-admin/visualization-tool#1604] [Q&A] [answered]
I can't find out how to change the language of a chart returned by the "Preview via API". 
There isn't anything indicating the language in the chart config JSON and adding the language path to the iframe URL (e.g. `<iframe src="https://test.visualize.admin.ch/de/preview"></iframe>`  doesn't change anything.

If there is currently no way to change the language it would be a useful feature in the future.

**Comment by bprusinowski** [comment]
Hi @sam-e-ch, that's indeed a good improvement idea, currently we explicitly set the language to "en" without a way to override this. I've added an issue to track this enhancement here: https://github.com/visualize-admin/visualization-tool/issues/1605

cc @sosiology

**Reply by Rdataflow** [reply]
@bprusinowski it would be wonderful if you could you takle this during tomorrow in case the effort fits as sam will leave us by the end of this week. Hope this works for you...

**Reply by bprusinowski** [reply]
@Rdataflow thanks for this information, I'll take a look tomorrow then, hopefully it's not a big effort 👍

**Reply by bprusinowski** [reply, marked as answer]
@sam-e-ch the feature should be available on TEST once the app deploys (in around 20 minutes). For the approach, you can see https://github.com/visualize-admin/visualization-tool/pull/1609 :)
